lib/components/popups/course-search/prereqs/PrereqDisplay.tsx

```diff
@@ -455,7 +455,12 @@ const PrereqDisplay: FC = () => {
         </>
       );
     else if (!loaded)
-      return <>{'Loading Prereqs Status: loaded is ' + loaded.toString()}</>;
+      return (
+        <>
+          <div>Loading Prereqs</div>
+          <img src="/img/loading.gif" alt="Searching..." className="h-7"></img>
+        </>
+      );
```

Comment:
awesome! looks better

Reply:
Just submitted PR in the backend repo :D. 
If you @mpark63 don't see any more issues guess we should be good to close this PR since the problem is with the db.

Reply:
yep looks good sorry for delay!